src/util/modules.py

```diff
@@ -486,8 +486,19 @@ def force_install_module(cr, module, if_installed=None):
     :param str module: name of the module to install
     :param list(str) or None if_installed: only force the install when these modules are
                                            already installed
-    :return str: the *original* state of the module
     """
+    if version_gte("saas~14.5"):
+        # We must delay until the modules actually exists. They are added by the auto discovery process.
+        if not if_installed or modules_installed(cr, *if_installed):
+            ENVIRON["__modules_auto_discovery_force_installs"].add(module)
```

Comment:
I would also the auto discovery fail when asked to force install an unknown module (if not already done)